app/components/request_form/request_form.css

```diff
@@ -8,9 +8,18 @@
 }
 
 .RequestForm-column + .RequestForm-column {
+    display: flex;
+    flex-direction: column;
```

Comment:
Hm, why only every item except the first? :thinking:

Reply:
Argh, good catch, that’s by mistake.